migrations/lib/importer/steps/user_custom_fields.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Migrations::Importer::Steps
+  class UserCustomFields < ::Migrations::Importer::CopyStep
+    depends_on :users, :user_field_values
```

Comment:
Does this step actually need the dependency on `user_field_values`?

Reply:
Not required , I started off without it. I added it later to cover the edge case where a `user_field_*` entry exists in `user_custom_fields` and the same user also happens to have a `user_field_values` record whose resolved name is the same `user_field_*`. In that situation, I wanted the `user_field_values` entry to take precedence. Is it unlikely? Yes, but still possible.